rest_framework/authtoken/serializers.py

```diff
@@ -18,13 +18,22 @@ def validate(self, attrs):
             if user:
                 if not user.is_active:
                     msg = _('User account is disabled.')
-                    raise exceptions.ValidationError(msg)
+                    raise exceptions.ValidationError(
+                        msg,
+                        code='authorization'
```

Comment:
Another solution would be to make the codes specific to the type of exception. eg.: "validation_error.authorization". This will also help clients that consume this API to unequivocally identify error codes/keys.